### ISSUE TITLE
Оставить отзыв

### ISSUE THREAD

**Issue by ro31337** [ro31337/rubyschool#5]
Дорогие друзья! Оставляйте ваши отзывы о руби школе. Понравились уроки? Нашли работу? Расскажите об этом! Большой отзыв? Без проблем! Сделать это просто:

* Пройдите по этой [ссылке](https://github.com/ro31337/rubyschool/blob/master/index.html) - там будет кнопка редактирования (карандаш).

* Отмотайте чуть вниз (примерно на 160 строку) и вставьте свой отзыв по аналогии с остальными отзывами. Вот вам шаблон для отзыва:

```html
          <p class="feedback">
            Ваш отзыв тут
            <br/>
            <i>&mdash; Ваша имя и фамилия (или псевдоним)</i>
          </p>
```

* Когда вы нажмете сохранить - Гитхаб сразу предложить создать Pull Request, прямо из браузера - соглашайтесь.

Не бойтесь сделать что-то не так, если что - я поправлю!

Заранее благодарю!

Р.

**Comment by sddw62** [comment]
Добрый день! А где можно найти сами уроки? Вижу только их описание. Заходил 3-го мая, вроде было видео, а сейчас нет.

**Comment by ro31337** [comment]
rubyschool.us

**Comment by sddw62** [comment]
спасибо!

**Comment by pestovpvl** [comment]
Отличный курс. Граммотное изложение материала. Прохожу с большим удовольсвием!
Спасибо, Роман!

**Comment by Shankshel** [comment]
Спасибо Вам за труд, все понятно и настолько оказывается просто, после просмотра Ваших уроков, сразу чувствуется профессионализм. Очень долго искал подробную информацию по Ruby и наконец нашел. Продолжайте пожалуйста в том же духе, хотелось бы от Вас получить информацию в точно таких же уроках по другим языкам, если это конечно возможно)

**Comment by ro31337** [comment]
Благодарю, @Shankshel Денис!

**Comment by hanuz06** [comment]
2019г..Материал уже устаревший же? И rails 4 скорее всего?

**Comment by ro31337** [comment]
@hanuz06 нет, материал не устаревший. Ruby не джаваскрипт ;) Все подробности в чате https://t.m/rubyschool

**Comment by ruslan-muffin** [comment]
очу поблагодарить тебя Роман! Мне было уже за 40 когда я начал проходить уроки в твоей руби школе. В последствии я прошел курсы по   Devops. Cейчас я девопс инженер в крупной компании и это после 40 лет. Сейчас мне 45. Поэтому учиться никогда не поздно.

**Comment by ValeraBro** [comment]
Привет)
Как жизнь?
За месяца 2 по вечерам возможно поднять руби до уровня выхода на работу?

**Comment by ro31337** [comment]
@ValeraBro привет, малореально

**Comment by ValeraBro** [comment]
> @ValeraBro привет, малореально

Бро, а в веб на руби возможно пойти с таким расчетом, чтобы найти потом работу ? Руби на данный момент на рынке растет или падает, я немогу найти данные по нему касательно рабочих мест вообще в целом)

**Comment by ValeraBro** [comment]
> @ValeraBro привет, малореально

Структура твоего курса очень хорошая, лично на мой взгляд. Но есть один момент, Который меня останавливает, если честно. Это то, что я не могу найти вакансий по РФ. Может я не так или не там ищу? Можешь подсказать с этим?

**Comment by ro31337** [comment]
Бро, давай в чат https://t.me/rubyschool

Я нашел работу 3 недели назад. Все остальные тоже чувствуют себя прекрасно. Неизвестны мне такие случаи кто стек сменил.

**Comment by ValeraBro** [comment]
Привет!) Я слышал, что у вас на прощёное воскресенье есть разбраны, это так?

**Comment by ValeraBro** [comment]
С наступающим новым годом, бро!

**Comment by ValeraBro** [comment]
Бро, ты говорил, что разбанишь 15 сентября. Разбана до сих пор нет, прям минус вайб пока что, сделай в чате анбан пжлст :3

**Comment by ValeraBro** [comment]
Дороу. Материалы актуальные? Говорят, что их уже сильно не достаточно для работы, это так?

**Comment by ValeraBro** [comment]
> rubyschool.us
Дороу. Материалы актуальные? Говорят, что их уже сильно не достаточно для работы, это так?

**Comment by ValeraBro** [comment]
Всем привет,нет апдейтов вот уже лет 6. Руби жив там?
